ui/components/ReconciledObjectsTable.tsx

```diff
@@ -137,8 +143,11 @@ function ReconciledObjectsTable({
 export default styled(ReconciledObjectsTable).attrs({
   className: ReconciledObjectsTable.name,
 })`
-  td:nth-child(5) {
+  td:nth-child(5),
+  td:nth-child(6) {
```

Comment:
These two selectors can be combined as a range (but your version is easier to read):
```
td:nth-child(n+5):nth-child(-n+6) {
    white-space: pre-wrap;
}
  ```